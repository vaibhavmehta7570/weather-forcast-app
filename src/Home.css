@import url(https://fonts.googleapis.com/css?family=Bungee);
.welcome-text {
  font-size: 5rem;
  font-family: 'Bungee';
}
