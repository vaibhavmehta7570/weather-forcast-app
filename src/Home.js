import React from 'react'
import background from './assets/home.svg'

const Home = () => {
  return <img className="home__background" src={background} alt="home" />
}

export default Home
