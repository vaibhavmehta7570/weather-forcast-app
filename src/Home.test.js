import React from 'react'
import Home from './Home'
import renderer from 'react-test-renderer'

describe('Should test Home.js', () => {
  test('Should render Home component correctly', () => {
    const HomeRendererResponse = renderer.create(<Home />).toJSON()
    expect(HomeRendererResponse).toMatchSnapshot()
  })
})
