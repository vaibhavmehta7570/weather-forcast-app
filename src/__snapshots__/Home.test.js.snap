// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should test Home.js Should render Home component correctly 1`] = `
<img
  alt="home"
  className="home__background"
  src="home.svg"
/>
`;
